bindings/python/src/torrent_info.cpp

```diff
@@ -307,6 +307,14 @@ void bind_torrent_info()
 
     class_<announce_entry>("announce_entry", init<std::string const&>())
         .def_readwrite("url", &announce_entry::url)
+        .def_readonly("trackerid", &announce_entry::trackerid)
+        .def_readonly("message", &announce_entry::message)
+        .def_readonly("last_error", &announce_entry::last_error)
```

Comment:
Most are `int` or `bool` so I wouldn't imagine there is an issue? Though `next_announce` and `min_announce` are `lt::time_point`.